docs/modules/publisher.md

```diff
@@ -121,24 +126,205 @@ A basic configuration file looks like this:
+| `OpcSamplingIntervalTimespan`    | No        | String  | null                      | The sampling interval for the monitored item to be <br>published. Value expressed in Timespan <br>string({d.hh:mm:dd.fff}). <br>Ignored when OpcSamplingInterval is present.  |
+| `OpcPublishingInterval`          | No        | Integer | 1000                      | The publishing interval for the monitored item to be <br>published. Value expressed in milliseconds. <br>This value will be overwritten when a publishing interval <br>is explicitly defined in the DataSetWriter owning this OpcNode.  |
+| `OpcPublishingIntervalTimespan`  | No        | String  | null                      | The publishing interval for the monitored item to be <br>published. Value expressed in Timespan <br>string({d.hh:mm:dd.fff}). <br>This value will be overwritten when a publishing interval <br>is explicitly defined in the DataSetWriter owning this OpcNode. <br>Ignored when OpcPublishingInterval is present  |
+| `DataSetFieldId`                 | No        | String  | null                      | A user defined tag used to identify the Field in the <br>DataSet telemetry message when publisher runs in <br>PubSub message mode.   |
+| `DisplayName`                    | No        | String  | null                      | A user defined tag to be added to the telemetry message <br>when publisher runs in Samples message mode.    |
+| `HeartbeatInterval`              | No        | Integer |    0                      | The interval used for the node to publish a value (a publisher <br>cached one) even if the value has not been <br>changed at the source. This value is represented in seconds. <br>0 means the heartbeat mechanism is disabled. <br>This value is ignored when HeartbeatIntervalTimespan is present      |
+| `HeartbeatIntervalTimespan`      | No        | String  | null                      | The interval used for the node to publish a value (a publisher <br>cached one) even if the value has not been <br>changed at the source. This value is represented in seconds. <br>Value expressed in Timespan string({d.hh:mm:dd.fff}).     |
+| `SkipFirst`                      | No        | Boolean | false                     | Instructs the publisher not to add to telemetry the<br> Initial DataChange (after subscription activation) for this OpcNode.     |
+| `QueueSize`                      | No        | Integer | 1                         | The desired QueueSize for the monitored item to be published.     |
+
+*Note*: **Id field may be omitted when ExpandedNodeIdId is present.
+
+A DataSet is a group of NodeIds within an OPC UA Server to be published with a same publishing interval.  
+DataSetWriter has one DataSet and contains the elements required to successfully establish a connection to the OPC UA Server. 
+A DatSetGroup is used to several DataSetWriters for a specific OPC UA server. 
```

Comment:
Is there a verb missing after `used to` ? Should it have been `used to group` ?

Reply:
done